qa/suites/rgw/s3tests.yaml

```diff
@@ -0,0 +1,10 @@
+# Each of the rgw subsuites that run s3-tests have a symlink to this file.
+# Changing the forece-branch on this file changes the branch of s3-tests run 
+# for all subsuites. S3tests config changes for each subsuite can be made in 
+# the subsuite's overrides.yaml or in s3tests specific configuration 
+# .yaml files in the subsuite.
+tasks:
+- s3tests:
+    client.0:
+      force-branch: ceph-master
+      rgw_server: client.0
```

Comment:
the base yaml shouldn't add the task, it should just override any existing tasks:
```yaml
overrides:
  s3tests:
    force-branch: ceph-master
    # git_remote: https://github.com/ceph/
```
if we put this in `qa/suites/rgw/s3tests.yaml`, teuthology will try to run it as a job under the rgw suite. let's put it under `qa/rgw/s3tests-branch.yaml` instead and symlink it from there

Reply:
> teuthology will try to run it as a job under the rgw suite

from your pulpito link, this job was https://pulpito.ceph.com/amaredia-2022-07-11_16:54:45-rgw-main-distro-default-smithi/6924096/

> Description: rgw/s3tests